tests/conftest.py

```diff
@@ -13,3 +15,18 @@ def client(app):
     client = app.test_client()
     client.testing = True
     return client
+
+
+@pytest.fixture(scope='function')
```

Comment:
that's the default scope - in other places we don't define it explicitly so I suggest we stick with that